server/chat-commands.js

```diff
@@ -4103,6 +4105,13 @@ const commands = {
 		}));
 	},
 
+	hidereplay(target, room, user, connection) {
+		if (!room || !room.battle || !this.can('joinbattle', null, room)) return;
```

Comment:
Should this also have a `forcedPublic` check?

Reply:
This is the command we specifically want OLT participants to have access to

Reply:
Please either reply or resolve conversations, but not both. When you do both, it hides your response from me.

Anyway, is there a link to discussion/reasoning?

Reply:
I left this comment unresolved for multiple review cycles after responding and your lack of activity on it made it seem resolved.

https://www.smogon.com/forums/threads/rated-games-and-modjoin.3652782/ is the thread. Below is confirmation from a TD:

```
16:38] pre: you wanted me to also steal /hidereplay from tours, right?
[16:40] Tony: let me double check if that's still what the other tds want
[16:41] Tony: ok yes we do
[16:41] Tony: if it's not too tricky to implement of course
[16:41] pre: Well, it's more a policy question
[16:42] Tony: i suggested it in the thread because i like the consistency it offers between smogon tour and olt qualification, while also giving the players some protection
[16:42] pre: Do we give it to all sim users? Just OLT prefixed users? Is it something we want to support at all etc?
[16:43] Tony: and the players responded positively to it, so did the tds
[16:43] Tony: well outside of olt is not really anything i have a say in i think
[16:43] Tony: it can't hurt right?
[16:43] Tony: if you have hideroom for non-olt games, then hidereplay seems harmless
```